superset-frontend/src/SqlLab/components/SqlEditor.jsx

```diff
@@ -417,6 +422,54 @@ class SqlEditor extends React.PureComponent {
     );
   }
 
+  renderDropdown() {
+    const qe = this.props.queryEditor;
+    const successful =
+      this.props.latestQuery && this.props.latestQuery.state === 'success';
+    const scheduleToolTip = successful
+      ? t('Schedule the query periodically')
+      : t('You must run the query successfully first');
+
+    return (
+      <Menu onClick={this.handleMenuClick}>
+        <Menu.Item>
+          {' '}
+          <span>Autocomplete</span>{' '}
+          <Switch
```

Comment:
are the spaces for styling? Can you use inline block style with margins instead?